parser/ParseCommand.cpp

```diff
@@ -0,0 +1,87 @@
+/**
+ *   Copyright 2016, Quickstep Research Group, Computer Sciences Department,
+ *     University of Wisconsin—Madison.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ **/
+
+#include "parser/ParseCommand.hpp"
+
+#include <cstdio>
+#include <map>
+#include <string>
+
+#include "parser/ParseString.hpp"
+#include "utility/StringUtil.hpp"
+
+#include "glog/logging.h"
+
+using std::string;
+using std::map;
+
+namespace quickstep {
+
+std::map<const std::string, std::string> ParseHelpCommand::help_messages_;
```

Comment:
The `key` field in the `map` marks `const` by default, so use `std::map<std::string, std::string> ...` instead.